projects/wallet-template/package.json

```diff
@@ -52,19 +52,21 @@
     "wxt": "^0.17.7"
   },
   "dependencies": {
+    "@headlessui/react": "^1.7.18",
     "@noble/ciphers": "^0.5.1",
     "@noble/hashes": "^1.4.0",
     "@polkadot-api/client": "0.0.1-492c132563ea6b40ae1fc5470dec4cd18768d182.1.0",
     "@polkadot-api/metadata-builders": "0.0.1-492c132563ea6b40ae1fc5470dec4cd18768d182.1.0",
     "@polkadot-api/substrate-client": "0.0.1-492c132563ea6b40ae1fc5470dec4cd18768d182.1.0",
     "@polkadot-api/tx-helper": "0.0.1-492c132563ea6b40ae1fc5470dec4cd18768d182.1.0",
     "@polkadot-api/utils": "0.0.1-492c132563ea6b40ae1fc5470dec4cd18768d182.1.0",
-    "@substrate/connect-known-chains": "workspace:^",
     "@polkadot-labs/hdkd": "^0.0.6",
     "@polkadot-labs/hdkd-helpers": "^0.0.5",
     "@react-rxjs/core": "^0.10.7",
     "@react-rxjs/utils": "^0.9.7",
+    "@std/ulid": "npm:@jsr/std__ulid@^0.220.1",
```

Comment:
let's use `crypto.randomUUID()` and not add an extra dependency

Reply:
Should we add a timestamp for when an account was created then? B/c we want to show the accounts in the ordered they were added. if we use a ulid we get ordering built in since its based on timestamp

Reply:
`Date.now()` should be fine.